modules/material/themes/material/default/selectidp-links.twig

```diff
@@ -28,10 +28,10 @@
   <header class="mdl-layout__header">
     <div class="mdl-layout__header-row">
       <span class="mdl-layout-title scale-to-parent">
-        {% if spName ?? '' is empty %}
+        {% if sp|entityDisplayName is empty %}
```

Comment:
Is the pipe correct here and below? Or should it be a `.`?

Reply:
It's correct as a filter. It is a custom filter defined by SimpleSAMLphp.